source/SupportLib/PlayGamesPluginSupport/src/main/java/com/google/games/bridge/PayloadCallbackProxy.java

```diff
@@ -0,0 +1,25 @@
+package com.google.games.bridge;
+
+import com.google.android.gms.nearby.connection.Payload;
+import com.google.android.gms.nearby.connection.PayloadCallback;
+import com.google.android.gms.nearby.connection.PayloadTransferUpdate;
+import android.support.annotation.NonNull;
+
+public class PayloadCallbackProxy extends PayloadCallback {
+	private Callback callback;
+
+    public PayloadCallbackProxy(Callback callback) {
+		this.callback = callback;
+	}
+
+	public void onPayloadReceived(@NonNull String endpointId, @NonNull Payload payload) {
```

Comment:
something with formatting

Reply:
it doesn't appear on my local repo for some reason. But manually updated it here.